packages/core/src/dispatcher.ts

```diff
@@ -49,6 +52,7 @@ export class Dispatcher {
     const pubsub = new Pubsub(_ipfs, topic, IPFS_RESUBSCRIBE_INTERVAL_DELAY, _pubsubLogger, _logger)
     this.messageBus = new MessageBus(pubsub)
     this.messageBus.subscribe(this.handleMessage.bind(this))
+    this.recordCache = new LRUMap<string, any>(IPFS_CACHE_SIZE)
```

Comment:
I guess the name "record" here is from when commits were called records. I would name this something like `objectCache` or `dagNodeCache`.

Reply:
I've been calling objects returned from the IPFS dag "ipfs records" - is that not proper terminology?